index.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap" rel="stylesheet">
+    <title>Lessons Learned</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+
+    <header id="cabecalho">
+        <h1 id="titulo">Lições Aprendidas - Trybe</h1>
+    </header>
+
+    <nav id="navegacao">
+        <ul>
+            <li>Home</li>
+            <li>Login</li>
+            <li><a href="https://developer.mozilla.org/pt-BR/" target="_blank">MDN - New window</a></li>
+        </ul>
+    </nav>
+
+    <main>
+        <img src="me.jpeg" alt="photo of matheus nugas" width="125px" id="minha_foto">
+        <div class="licoes">
+            <ol id="licoes_aprendidas">
+                <h2>Lições Aprendidas</h2>
+                <li>Soft skills</li>
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>HTML Semantico</li>
+                <li>Unix</li>
+                <li>Bash</li>
+                <li>Git</li>
+                <li>GitHub</li>
+                <li>Terminal</li>
+                <li>NPM</li>
+            </ol>
+            <ul id="licoes_a_aprender">
+                <h2>Lições a Aprender</h2>
+                <li>Redux</li>
+                <li>JavaScript</li>
+                <li>React</li>
+                <li>Python</li>
+                <li>MySQL</li>
+                <li>MongoDB</li>
+                <li>DOM</li>
+                <li>Jest</li>
+                <li>SCRUM</li>
+                <li>NodeJS</li>
+            </ul>
+        </div>
+
+        <section class="artigos">
+            <article class="lado-esquerdo">
+                <h2>Como foi até agora</h2>
+                <p>
+                    Lorem, ipsum dolor sit amet consectetur adipisicing elit. Voluptatem obcaecati consectetur quos
+                    dolore
+                    voluptas iste, voluptate consequuntur officiis dolores illo hic dolorem aspernatur enim doloribus
+                    ipsam
+                    facere adipisci debitis incidunt! Odio sequi vitae accusamus a in animi quos vel, doloremque nobis
+                    quo.
+                </p>
+            </article>
+
+            <aside class="lado-direito">
+                <h2>Expectativas para o futuro</h2>
+                <p>
+                    Lorem ipsum dolor sit amet consectetur adipisicing elit. Quos non sed tempora voluptatem asperiores,
+                    quod fugiat dolor perspiciatis, porro quae tempores.
+                </p>
+            </aside>
+        </section>
+
+        <table border="1">
```

Comment:
Tenta sempre separar estilo de estrutura. Nesse caso, cria a borda da tabela no arquivo css. 
```suggestion
        <table>
```